<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlateManipulatorLayout</class>
 <widget class="QWidget" name="PlateManipulatorLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>721</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plate widget</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="barcode_label">
         <property name="text">
          <string>Plate barcode: </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="barcode_ledit">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text">
          <string>TTP3000807</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="sample_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Double click to move to position in the plate</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="numRows" stdset="0">
          <number>0</number>
         </property>
         <property name="numCols" stdset="0">
          <number>0</number>
         </property>
         <property name="readOnly" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="navigation_graphicsview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="current_location_label">
         <property name="text">
          <string>Current location: </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="current_location_ledit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="xtal_treewidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>400</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <column>
        <property name="text">
         <string>Element</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Label</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Login</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Row</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Col</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comment</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="move_button">
         <property name="text">
          <string>Move to crystal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="reposition_cbox">
         <property name="text">
          <string>Use crystal repositioning</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="abort_button">
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="xtal_image_graphicsview">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
